lib/model/rwfolder.go

```diff
@@ -305,6 +213,99 @@ func (f *sendReceiveFolder) String() string {
 	return fmt.Sprintf("sendReceiveFolder/%s@%p", f.folderID, f)
 }
 
+func (f *sendReceiveFolder) pull(prevSec int64, prevIgnoreHash string) (int64, string, bool) {
+	select {
+	case <-f.initialScanFinished:
+	default:
+		// Once the initial scan finished, a pull will be scheduled
+		return prevSec, prevIgnoreHash, true
+	}
+
+	f.model.fmut.RLock()
+	curIgnores := f.model.folderIgnores[f.folderID]
+	f.model.fmut.RUnlock()
+
+	if newHash := curIgnores.Hash(); newHash != prevIgnoreHash {
```

Comment:
Can I ask to have the variable naming tweaked a bit? This is not your fault, it's historical, but might as well fix it while we're anyway poking round.

 - The method signature here is complex enough (three return values) that I think it would add clarity to name the return values. I *don't* suggest using bare returns or anything - just giving the return values names so it's clear what they are from looking at the method signature.

- The `prevSec` variable/parameter is misspelled, it should be `prevSeq` as it's about a sequence number and not a time or something.

- The method *takes* `prevSeq` and `prevIgnoreHash` but the values it calculates and compares them to in the method should be `curSeq` and `curIgnoreHash`, and those should be what it returns. It's confusing to return `prevSeq` etc at the end of the method, as if those were the same things we got in.

Reply:
Done. The `curSeq` feels a bit weird, as it is resulted in `curSeq = prevSeq`, 's/curSeq/remoteSeq/g` and 's/prevSeq/curSeq/g` - but I guess it is clearer.